README.md

```diff
@@ -1,4 +1,27 @@
 # visual-programming
 
-### Initial Prerequisites
-`pip install -r requirements.txt`
+So far the app comprises a Django app and a SPA React app (bootstrapped with
+create-react-app). For React to request data from Django, the `proxy` field is
+set in `front-end/package.json`, telling the dev server to fetch non-static
+data from `localhost:8000` **where the Django app must be running**.
+
+## Django
+
+**Install Prerequisites**
+- `pip install -r requirements.txt`
+
+**Create dotenv file with app secret**
+- `echo "SECRET_KEY='TEMPORARY SECRET KEY'" > .environment`
```

Comment:
I've always had a single .env file at the root-level, so ran into a brief issue here. Fixed by moving the file to `vp/.environment` which is probably as intended.

Reply:
Good call, fixed README in https://github.com/matthew-t-smith/visual-programming/pull/3/commits/481a15eb926410921bf15fff1e8687919ded59a2